lib/matplotlib/legend.py

```diff
@@ -978,13 +986,16 @@ def _get_anchored_bbox(self, loc, bbox, parentbbox, renderer):
         Place the *bbox* inside the *parentbbox* according to a given
         location code. Return the (x, y) coordinate of the bbox.
 
-        - loc: a location code in range(1, 11).
-          This corresponds to the possible values for self._loc, excluding
-          "best".
+        Parameters
+        ----------
+        loc : int
+            A location code in range(1, 11). This corresponds to the possible
+            values for self._loc, excluding "best".
+        bbox : `~matplotlib.transforms.Bbox`
+            bbox to be placed, display coodinate units.
```

Comment:
coodinate: typo
also be consistent with immediately below (e.g. ", in display coordinates.") for both.